### ISSUE TITLE
Does trt 8.0 support Bidirectional LSTM  ?

### ISSUE THREAD

**Issue by cding-nv** [NVIDIA/TensorRT#1417]
Does trt 8.0 support Bidirectional LSTM  ?

```
from keras.layers import Bidirectional
from keras.layers.recurrent import LSTM

x = Bidirectional(LSTM(32, return_sequences=True))(x)

```
Thanks

**Comment by cding-nv** [comment]
$ trtexec --onnx=./bidirectional_LSTM_.onnx --saveEngine=my.engine
 gets the below error
```
[08/02/2021-06:12:13] [E] [TRT] ModelImporter.cpp:723: --- End node ---
[08/02/2021-06:12:13] [E] [TRT] ModelImporter.cpp:725: ERROR: builtin_op_importers.cpp:2339 In function importLSTM:
[8] Assertion failed: std::equal(activationAlphas.begin(), activationAlphas.begin() + NUM_ACTIVATIONS, activationAlphas.begin() + NUM_ACTIVATIONS) && "The parser does not currently support cases where activations for the reverse pass of the LSTM do not match the forward pass."
[08/02/2021-06:12:13] [E] Failed to parse onnx file
[08/02/2021-06:12:13] [I] Finish parsing network model
[08/02/2021-06:12:13] [E] Parsing model failed
[08/02/2021-06:12:13] [E] Engine creation failed
[08/02/2021-06:12:13] [E] Engine set up failed
```

**Comment by cding-nv** [comment]
Where is the "CgPersistentLSTMPlugin_TRT" in this link https://docs.nvidia.com/deeplearning/tensorrt/developer-guide/index.html ->**4.4.4. Persistent LSTM Plugin**

**Comment by cding-nv** [comment]
The issue has been fixed:   The keras version  is upgraded from 2.2.4 to 2.3.1

**Comment by ttyio** [comment]
cool to hear it is fixed, closing

**Comment by haiderasad** [comment]
> The issue has been fixed: The keras version is upgraded from 2.2.4 to 2.3.1



> The issue has been fixed: The keras version is upgraded from 2.2.4 to 2.3.1

meaning when you created model in keras 2.3 the engine creation worked?
